Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/DivXmlElementAccess.java

```diff
@@ -257,9 +257,13 @@ DivType toDiv(Map<PhysicalDivision, String> physicalDivisionIDs, LinkedList<Pair
         }
         div.setORDERLABEL(super.getOrderlabel());
         div.setTYPE(super.getType());
-        smLinkData.addAll(super.getViews().stream().map(View::getPhysicalDivision)
-            .sorted(Comparator.comparing(PhysicalDivision::getOrder)).map(physicalDivisionIDs::get)
-            .map(physicalDivisionId -> Pair.of(metsReferrerId, physicalDivisionId)).collect(Collectors.toList()));
+        if (!super.getViews().isEmpty()) {
+            smLinkData.addAll(super.getViews().stream().map(View::getPhysicalDivision)
+                    .sorted(Comparator.comparing(PhysicalDivision::getOrder)).map(physicalDivisionIDs::get)
+                    .filter(Objects::nonNull)
```

Comment:
I suppose this or the emptiness check is the actual critical fix: Preventing that a physical division which is null at that point gets added to the SMLink section. I have no objections here. 
You probably have to save a second time to have a complete link section then.